x-pack/plugins/reporting/export_types/csv/server/execute_job.js

```diff
@@ -33,9 +34,10 @@ function executeJobFn(server) {
     try {
       decryptedHeaders = await crypto.decrypt(serializedEncryptedHeaders);
     } catch (e) {
-      throw new Error(
-        'Failed to decrypt report job data. Please ensure that xpack.reporting.encryptionKey is set and re-generate this report.'
-      );
+      throw new Error(i18n.translate('xpack.reporting.exportTypes.csv.executeJob.failedToDecryptErrorMessage', {
```

Comment:
failedToDecryptErrorMessage => failedToDecryptReportJobDataErrorMessage
I believe ids should be a little bit more descriptive